### ISSUE TITLE
Refresh tab does not release memory 

### ISSUE THREAD

**Issue by inear** [brave/browser-laptop#13273]
I have been following an issue in chromium in the current version of Chrome (M64) that have problems with refreshing tabs on pages that use WebAudio. V8 keeps references to older contexts because of a memory leak. The same problem can be seen in Brave. It's not happening in M65 so it should be fine here as well once it's reaches Brave. (I have novice understanding about open source and release-cycles etc) Just want to mention it here because it can possible be the cause of some of the "sluggish performance" issues.

### Steps to Reproduce
Go to a page, preferable a html5 game or 3d page using WebAudio. Inspect performance monitor or profile JS heap and see memory footprint continue to increase. Change domain and memory is released.

**Actual result:**
Memory keeps increasing for every refresh.

**Expected result:**
Memory should be released.

### Brave Version 
Brave: 0.20.42 
V8: 6.4.388.41 
rev: 096c7cb3c75ebb518f72fc7d24bc2bbaedd50aed 
Muon: 4.8.2 
OS Release: 16.6.0 
Update Channel: Release 
OS Architecture: x64 
OS Platform: macOS 
Node.js: 7.9.0 
Brave Sync: v1.4.2 
libchromiumcontent: 64.0.3282.140

**Reproducible on current live release:**
Yes

### Additional Information
https://bugs.chromium.org/p/chromium/issues/detail?id=810701#c29

**Comment by btlechowski** [comment]
Confirmed 0.20.42

Thank you for reporting the issue. Indeed Brave will update Chromium to 65. We will keep the issue open till the new Chromium is integrated.

**Comment by hferreiro** [comment]
I have just verified that Brave on C65 does not have this issue by trying to reproduce the steps in https://bugs.chromium.org/p/chromium/issues/detail?id=810701#c11

**Comment by bsclifton** [comment]
Great, thanks @hferreiro! I'll close this issue.  Can you share steps you tried? When I follow the link you shared, it mentions the Chrome Task Manager. How did you verify the memory? (just viewing Activity Monitor or something like this?)

**Comment by srirambv** [comment]
I can't make out any change in resource utilization using the resource monitor on Windows when I visit http://luminance.org/stuff/810701/audio-leak.html and follow the steps mentioned in the original chromium bug. @hferreiro Any specific steps to be performed on Windows to verify this?

**Comment by hferreiro** [comment]
@bsclifton I tested on Linux and used Gnome System Monitor. Any tool that shows memory usage by process should be ok, I guess.

**Comment by LaurenWags** [comment]
Reload of the page didn't really seem to do anything but Clean Reload freed memory (went from about 214.x MB to 171.x MB), is this sufficient @bsclifton @hferreiro ?  cc @kjozwiak

**Comment by btlechowski** [comment]
The reload button does not free the memory. I used F5 and it worked. One of the processes grew to 90MB, then shrank after reload(F5) back to 20MB.

Verified on Win7 v0.21.19.

Steps followed:
1. Launch Browser
2. Open Chrome task manager
3. Navigate to http://luminance.org/stuff/810701/audio-leak.html
4. Click on issue an xhr, then click decode, then click play (Do the same for x times) - grew around 20MB for every iteration.
5. Refresh the page
